docs/Config.md

```diff
@@ -53,6 +53,9 @@ oS:
   openCommand: open {{filename}}
   openLinkCommand: open {{link}}
 update:
+  refreshProjectTime: 100ms
+  refreshContainersAndServicesTime: 100ms
+  refreshVolumesTime: 100ms
   method: never
```

Comment:
may as well remove this line given that we're not going to use it

Reply:
Why remove this if md file shows all config items (to my understanding).  
this file is easy to understand for new users or for users that want to quickly know if something explicit can be configured, this way they they don't have to search inside the godoc for that item.

Reply:
sorry what I meant was: we're not using this 'method' key anywhere in the app. This is a vestige from lazygit where you could enable regular auto-updates (as in installing the latest lazygit), but we're not using it here so there's no harm in deleting it given that the context of the 'update' config key has changed from installations to refresh rates.

Reply:
Fixed it